resinos-in-container.sh

```diff
@@ -116,11 +116,20 @@ resin_state_volume="resin-state-$container_id:/mnt/state"
 resin_data_volume="resin-data-$container_id:/mnt/data"
 
 # Populate the boot volume with the config.json
-docker run -i --rm -v \
-	"$resin_boot_volume" -v "$config_json":/config.json \
+if [ -n "$config_json" ]; then
+	populate_extra_args="-v $config_json:/config.json"
+fi
+docker run -i --rm \
+	-v "/usr/bin/qemu-aarch64-static:/usr/bin/qemu-aarch64-static:ro" \
```

Comment:
Why do we need qemu-aarch64-static here?

Reply:
It is a left over. Will remove before pushing last version.